webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/NodeGroupCategoryForm.scala

```diff
@@ -149,12 +150,26 @@ class NodeGroupCategoryForm(
            (
              "input [disabled]" #> "true"
              & "textarea [disabled]" #> "true"
+             & "directive-close" #>
+             <button class="btn btn-default" onclick={
+               s"""$$('#${htmlIdCategory}').trigger("group-close-detail")"""
+             }>
+               Close
+               <i class="fa fa-times"></i>
+             </button>
              & "directive-save" #> SHtml.ajaxSubmit("Update", onSubmit _, ("class", "btn btn-success"))
              & "directive-delete" #> deleteButton
            )
          } else
            (
-             "directive-save" #> (
+             "directive-close" #>
```

Comment:
I think this could have been regrouped outside of the if (it does not depend on the category to be system or not)

Reply:
Done (after the approval but before the merge, sorry I forgot to make the change) : https://github.com/Normation/rudder/pull/5498/commits/973900200b18f1bd313ad671970373ba15e83a76